news/2021-05-07-monthly-beatmapping-contest-may-2021.md

```diff
@@ -0,0 +1,112 @@
+
+**[osu!mania MBC](LINK) limitation:**
+
+- Long notes cannot be used.
+
+### <a id="songs"></a>Songs
+
+Each MBC provides beatmappers a choice between songs from one of osu!'s [Featured Artists](https://osu.ppy.sh/beatmaps/artists).
+
+This month's artist is guitar savant [**Masahiro "Godspeed" Aoki**](https://osu.ppy.sh/beatmaps/artists/121). These three songs are available for mapping in the osu!, osu!taiko, and osu!mania contests:
+
+#### Frostbite (188 BPM)
+
+Beatmap template download: [Short version](LINK) (1:47) / [Full version](https://assets.ppy.sh/artists/136/Veiled/Masahiro%20Godspeed%20Aoki%20-%20Frostbite.mp3) (3:49)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/136/Veiled/Masahiro%20Godspeed%20Aoki%20-%20Frostbite.mp3">
+</audio>
+
+#### The Game Changer (280 BPM)
+
+Beatmap template download: [Short version](LINK) (2:24) / [Full version](https://assets.ppy.sh/artists/136/Songs/Masahiro%20Godspeed%20Aoki%20-%20The%20Game%20Changer.mp3) (3:44)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/136/Songs/Masahiro%20Godspeed%20Aoki%20-%20The%20Game%20Changer.mp3" type="audio/mpeg">
+</audio>
+
+#### busy come, busy go (instrumental version) (270 BPM)
+
+Beatmap template download: [Short version](https://assets.ppy.sh/artists/136/The%20Game%20Changer/Masahiro%20Godspeed%20Aoki%20-%20busy%20come,%20busy%20go%20(instrumental%20version).osz) (2:15)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/136/The%20Game%20Changer/Masahiro%20Godspeed%20Aoki%20-%20busy%20come,%20busy%20go%20(instrumental%20version).mp3" type="audio/mpeg">
+</audio>
+
+## Rules
+
+Breaking any of these rules will land you in MBC prison:
+
+- **follow the month's [beatmapping limitation](#limitation) for the contest's mode:**
+    - **osu!:**
+        - Total slider velocity cannot be higher than 1.4
+    - **osu!taiko:**
+        - All patterns on snaps faster than 1/2 must have an **even** number of notes.
+    - **osu!mania:**
+        - Long notes cannot be used.
+- **be in `.osz` format.** Including a background image and custom hitsounds is fine, but skin/storyboard/video content will not be considered.
+- **be a single difficulty for one of the month's [selected songs](#songs).**
+- **not be uploaded to the osu! website through the Beatmap Submission System (*BSS*).** All contest submissions are anonymous during the screening/judging phases to avoid bias. **DO NOT SHARE YOUR ENTRY!**
+- **be solely created by you.** Collaborations are not allowed.
+- **abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes, submissions that could not be ranked without major changes will be disqualified.
+
+## Submission
+
+You've got a bit under **3 weeks** to prepare your masterpiece of a map! Once it's ready, head over to the relevant contest listing to submit:
```

Comment:
```suggestion
You've got a bit under **3 weeks** to prepare your masterpiece of a map! Once it's ready, head on over to the relevant contest listing to submit:
```
nitpick

Reply:
This is not required. Where are you getting these suggestions from? If you're going to propose english changes please link a source showing the previous version is not correct, as most of the changes you are proposing are in fact already valid english.

Reply:
It's a nitpick, not a major issue. Imo it sounds better, but it's up to the writer to decide if they think so as well

Reply:
ill keep it as is